moped-editor/src/components/GridTable/SearchBar.js

```diff
@@ -177,13 +179,15 @@ const SearchBar = ({
     }
   };
 
-  const filterStateActive = !!Object.keys(filters).length;
-  const filtersApplied = [];
-  if (filterStateActive) {
-    Object.keys(filters).map((parameter) =>
-      filtersApplied.push(filters[parameter]["label"])
-    );
-  }
+  const filterStateActive = filters.length > 0;
+  const filtersApplied =
+    filterStateActive &&
+    filters.map((filter) => {
+      const fieldFilterConfig = filtersConfig.fields.find(
```

Comment:
We have to grab the filter name to display since we don't have it in state anymore. Not sure how I feel about having to use `find()` here, but changing the shape of `PROJECT_LIST_VIEW_FILTERS_CONFIG.fields` from an array to an object feels like a big change to me. We don't have that many fields to iterate but I'm not writing it off completely.